service/conf/application.conf

```diff
@@ -93,3 +93,47 @@ application.global=util.Global
 play.http.filters=util.Filters
 play.http.parser.maxMemoryBuffer = 900000K
 play.http.parser.maxDiskBuffer = 900000K
+
+
```

Comment:
Add a simple comment to demarcate the registry client configuration.

# Open Saber (Registry) Client Specific config
# ~~~~~

Reply:
Review comment incorporated in the config file